<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewObject</class>
 <widget class="QDialog" name="NewObject">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>166</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>166</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>166</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="Name_lbl">
       <property name="text">
        <string>Имя</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="Name_line"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="X_lbl">
       <property name="text">
        <string>Координата X</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="X_line"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="Y_lbl">
       <property name="text">
        <string>Координата Y</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="Y_line"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="Type_lbl">
       <property name="text">
        <string>Тип</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="Type_box">
       <item>
        <property name="text">
         <string>Нет</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Человек</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Здание</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Машина</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Дерево</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="OK_btn">
       <property name="text">
        <string>ОК</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Cancel_btn">
       <property name="text">
        <string>Отмена</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
